.github/workflows/nightly-build.yml

```diff
@@ -23,6 +24,9 @@ jobs:
         name: Build and push
         uses: docker/build-push-action@v2
         with:
-          context: openshift/system
+          context: .
+          file: openshift/system/Dockerfile
           push: true
-          tags: quay.io/3scale/porta:nightly
+          tags: |
+            quay.io/3scale/porta:latest
```

Comment:
Again just to consider, could this confuse anybody trying to pull latest stable build but get latest CI build?

Reply:
What do you mean? `latest` is the build of the default branch

Reply:
In priciple - yes. As a user though, I usually expect when I do `podman pull fedora` to bring me latest stable fedora release, not a nightly.